### DISCUSSION TITLE
Backup and Restore

### DISCUSSION THREAD

**Discussion by blackship1** [teslamate-org/teslamate#2557] [Q&A]
Hallo Adrian,

ich teslamate (docker) erfolgreich auf einem raspi 3 laufen. Nun möchte ich auf eine Proxmox Umgebung wechseln. Aber irgendwie klappt das mit dem Backup/Restore nicht. Außerdem verwende ich den gleiche Token auf der neuen Umgebung. Auf dem Port 4000 sehe ich den Stnadort, Km und so; aber es werden keine Daten gesammelt :-( 

Hat irgend eine Lösung für mich?

Gruß
Mike

**Comment by martinh2011** [comment]
There might be someone with a solution to your problem, but that needs more information than your „it does not work“. How did you make the backup and restore? Do you get any errors? What does the postgresql container log show (docker-compose logs database)?

Regarding new data: What does the Teslamate comtainer log show (docker-compose logs teslamate)?

**Comment by blackship1** [comment]
Hi, 

 

many thanks for your reply and sorry for my late answer.

But now (today), I solved the problem, there might be a problem with the credentials teslamate and Grafana from the backup  I used.

 

I installed everything new and used the ”docker-compose.yml” datei from the old Installation, and then everything works fine.

Backup from the old system and restore to the new proxmox destination 😊

 

I hope I can contact you, when I have new problems.

 

Have a nice weekend

Mike

 

 

Von: Martin Hüser ***@***.***> 
Gesendet: Montag, 28. März 2022 07:32
An: adriankumpf/teslamate ***@***.***>
Cc: blackship1 ***@***.***>; Author ***@***.***>
Betreff: Re: [adriankumpf/teslamate] Backup and Restore (Discussion #2557)

 

There might be someone with a solution to your problem, but that needs more information than your „it does not work“. How did you make the backup and restore? Do you get any errors? What does the postgresql container log show (docker-compose logs database)?

Regarding new data: What does the Teslamate comtainer log show (docker-compose logs teslamate)?

**Reply by degsec** [reply]
Hi Mike,
ich habe genau das selbe vor wie du. Sprich TeslaMate auf eine neue Proxmox Umgebung umzuziehen. Einziger Unterschied ist, dass ich Portainer nutze (sowohl auf dem Raspi, also auch in der Proxmox Umgebung).

Ich finde die "Backup & Restore" Anleitung etwas minimalistisch und stehe etwas aufm Schlauch. Kannst du vielleicht erklären, wie du das erfolgreich umgesetzt hast?

Danke im Voraus!

**Reply by blackship1** [reply]
Hallo,

also ich habe es genau nach dieser Anleitung durchgeführt https://docs.teslamate.org/docs/maintenance/backup_restore. Ich glaube entscheidend war, dass ich bei der installation auf die neue Umgebung die alte yml-Datei genutzt habe, wie oben schon beschrieben. Nach der neuen Installation habe ich ein backup+restore von der neuen Installation durchgeführt (zum testen ob es in beide Richtungen funktioniert) und dann die backup-datei der alten Installation an die neue Stelle kopiert.
